content/13.conclusion.md

```diff
@@ -1,7 +1,6 @@
 ## Conclusion {#conclusion}
 
 Deep learning techniques have the potential for wide use in biology, meeting or exceeding the performance of both humans and the current state-of-the art algorithms in a variety of tasks.
-Beyond simply achieving performance, deep learning has the potential to generate novel biological insights that may then be applied back to fundamental research.
+Beyond simply achieving good predictive performance, deep learning has the potential to generate novel biological insights that may then be applied back to fundamental research.
```

Comment:
> biological insights that may then be applied back to fundamental research.

Another little suggestion as this sounds a bit awkward: how about changing this to 

> biological insights that can assist the progress of fundamental research.